content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md

```diff
@@ -0,0 +1,86 @@
+
+- **Inconsistent or overly complicated code** - code should be recognisable and readable by all engineers.
+- **Design shortcuts** - align with your architecture, operations, and development patterns and guardrails. 
+- **Skipped or insufficient testing** - if you are strapped into your seat, atop a Falcon Heavy rocket using your software, you should feel comfortable to click the launch button.
+- **Outdated or end-of-life products** - proactively monitor and deprecate outdated dependencies.
+- **Incomplete or no documentation** - code should be self-explanatory, complemented by documentation that makes it easy to get the context,  overview, and flow. 
+
+Stop supporting the notion of "quick experiment", "proof-of-concept", or "throw-away" code. I have watched too many instances of this code ending up in production solutions and becoming the weakest link. Use common guidelines, guardrails, and peer-review workflows (such as pull request with policies) to avoid your solution starting to rust and its quality to deteriorate over time.
+
+# Think team!
+
+We are accountable for QUALITY as a team and as a group of teams - in other words, we "are one" when it comes to quality and taking responsibility for how we design, build, test, and maintain the solutions we build. As **software development engineer** our specialty is "developing code" and as **quality assurance engineer** our specialty is "testing code”. 
+
+> ![<SAMPLE PIC>](../images/embedding-a-quality-driven-engineering-mindset-in-our-dna-1.png) 
+
+However, this does not imply that developers are only tasked with writing code. They also shoulder responsibilities such as creating unit tests, executing test automation, performing code vulnerability scans, and in some cases, conducting manual tests to enhance the overall quality of our work. As engineers, we hold a stewardship role in our domain of expertise, enforcing associated guardrails, and mentoring others within our team, group, organization, or community. But, when it comes to quality, it is a collective team responsibility!
+
+Gone are the days of simply tossing tasks over the wall to different roles like content editors, developers, testers, admins, and so forth. The only boundary we acknowledge are the faintly dotted lines that delineate teams and groups of teams within our organization. Beyond that, we operate and own QUALITY as one unified entity.
+
+# Think product team!
+
+When you operate with a healthy [DevOps](https://www.donovanbrown.com/post/what-is-devopss) mindset, you will also be familiar with the "product mindset", which implies that we, as a team, are accountable for our solution from ideation to deprecation. Until we switch off the solution lights, we own the accountability for QUALITY!
+
```

Comment:
Pride, feeling like you've created and contributed to something you are proud of - that is the inner spirit we need to kindle.